react-client/src/containers/Login/Login.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import style from './Login.module.scss'
+import { useState, useEffect } from 'react'
+import UsernameIcon from '../../components/Common/Login/UsernameIcon/UsernameIcon'
+import PasswordIcon from '../../components/Common/Login/PasswordIcon/PasswordIcon'
+import { submitLogin } from '../../services/Podcasts'
+import { useNavigate } from 'react-router-dom'
+
+const Login = () => {
+  const [loginDetails, setLoginDetails] = useState('')
+  const [loginStatus, setLoginStatus] = useState('')
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    document.body.style.background = 'linear-gradient(to right, #11998e 0%, #38ef7d 100%)'
+    document.body.style.color = '#074d47'
+    return () => {
+      document.body.style.background = '#093637'
+      document.body.style.color = 'rgba(0, 241, 255, 0.84)'
+    }
+  }, [])
+
+  function handleChange(event) {
+    const { name, value } = event.target
+    setLoginDetails(prevValue => {
+      return {
+        ...prevValue,
+        [name]: value
+      }
+    })
+  }
+
+  async function submitLoginDetails() {
+    try {
+      window.authorization = await submitLogin(loginDetails)
+      setLoginStatus('')
```

Comment:
it's not the best practice to put a ui message as state data and call it status. it doesn't represent a status. it represent a message.
I would add to state a boolean called isLoginError  with a default of false. when set to true- the ui will show the error message of "Incorrect username or password" (an element that this is it's text)

Reply:
Using simple boolean doesn't re-render the element so I used a useState boolean